test/common/common.go

```diff
@@ -96,21 +96,8 @@ type MemcacheConfig struct {
 	Port int
 }
 
-// startCacheProcess starts memcache or redis as a subprocess and waits until the TCP port is open.
-func startCacheProcess(ctx context.Context, command string, args []string, port int) (context.CancelFunc, error) {
-	ctx, cancel := context.WithCancel(ctx)
-	cmd := exec.CommandContext(ctx, command, args...)
-
-	cmd.Stdout = os.Stdout
```

Comment:
This is going to have a merge conflict with my other diff to fix the hangs on panic()

Reply:
Merged